cellxgene_schema_cli/scripts/gene_processing.py

```diff
@@ -248,13 +253,29 @@ def process_gene_infos(self, gene_infos: dict) -> None:
                     )
                 self.write_gzip(output_to_print, new_file)
 
-                if self.digest(new_file) == self.digest(previous_ref_filepath):
-                    print("New gene reference is identical to previous gene reference", gene_info_description)
-                    os.remove(new_file)
-                else:
-                    print("generating gene reference diff for", gene_info_description)
-                    self.generate_gene_ref_diff(gene_info_description, new_file, previous_ref_filepath)
+                # Rename new_genes_{filename} to genes_{filename} if this is the first time we're introducing a new gene file type
```

Comment:
the way the script is currently written is that it expects you to have a previous version to reference. i've updated it to account for if we're introducing a new gencode file type for the first time by:
- renaming `new_genes_fruit_fly.csv.gz` to `genes_fruit_fly.csv.gz` and skipping over the check against the previous version, since there is no previous version to reference
- generating a `fruit_fly_diff.txt` file which includes all of the new genes we're introducing